swan-cern-system/templates/fluentd/fluentd_sources.conf.yaml

```diff
@@ -12,9 +12,19 @@ data:
 
       {{ if not .Values.fluentd.output.includeInternal }}
       <match kubernetes.var.log.containers.**_kube-system_**>
+        {{ if not .Values.fluentd.output.includeFalco }}
           @type null
+        {{ else }}
+          <filter kubernetes.var.log.containers.**_kube-system_**>
+            @type grep
+            <regexp>
+              key $.kubernetes.pod_name
```

Comment:
So this system makes sure that ONLY the logs of this Falco pod go through, and the rest of the logs of all the remaining pods in kube-system are discarded?